internal/logic/logic.go

```diff
@@ -221,7 +219,6 @@ func (l userMgmtSvcLogic) Activate(id any) *respModel.Response {
 }
 
 func (l userMgmtSvcLogic) UserData(id any) *respModel.Response {
```

Comment:
Why is any used as the argument? This should be a string.

Reply:
done